CHANGELOG.md

```diff
@@ -41,3 +41,6 @@
 
```

Comment:
Add line `<!--next-version-placeholder-->` here

Reply:
Like this? :)